.github/workflows/ci.yml

```diff
@@ -7,6 +7,9 @@ jobs:
     steps:
       - uses: actions/checkout@v4.1.2
 
+      - name: Install Nushell
+        run: cargo install nu --locked --no-default-features
+
```

Comment:
Install Nu by this way is quite slow, try this one: https://github.com/hustcer/setup-nu